packages/opentelemetry-core/src/utils/trace-flags.ts

```diff
@@ -0,0 +1,26 @@
+/*!
```

Comment:
I think we should rename file to something else, developers would get confused with `trace_flags.ts` which is in API.

Reply:
This function was going to be used in span processors, but now that it isn't it is only used in a single place. I'll just remove the file completely.